scripts/generate_access_token.sh

```diff
@@ -66,7 +66,7 @@ if [[ $GHA_SECRET == "local_dev" ]]; then
     cp $SERVICE_ACCOUNT ~/.credentials/
 else
     echo "Decrypting ${SERVICE_ACCOUNT_FILE}.gpg"
-    scripts/decrypt_GHA_SECRET.sh $SERVICE_ACCOUNT ~/.credentials/$SERVICE_ACCOUNT_FILE "$plist_secret"
```

Comment:
This shouldn't fix the nightly test errors seen in #9086 but are improvements I made while trying to reproduce nightly failures.